front/app/containers/Admin/projects/project/analysis/Tags/index.tsx

```diff
@@ -164,55 +178,65 @@ const Tags = () => {
           onCloseModal={() => setAutotaggingModalIsOpened(false)}
         />
       </Modal>
-      <Box>
-        <Button
-          id="auto-tag-button"
-          onClick={() => setAutotaggingModalIsOpened(true)}
-          icon="flash"
-          mb="12px"
-          size="s"
-          buttonStyle="admin-dark"
-        >
-          {formatMessage(translations.autoTag)}
-          {emptyState && (
-            <BlickingIcon
-              name={'dot'}
-              width="16px"
-              height="16px"
-              fill={colors.white}
-              ml="8px"
+      <Box
+        position="fixed"
```

Comment:
The problem with the previous approach was that when you open the three dots menu, it got hidden behind the sticky controls on top (the button and create tag input). This new positioning solves the issue.